toolchain/parse/handle_binding_pattern.cpp

```diff
@@ -51,7 +51,8 @@ auto HandleBindingPattern(Context& context) -> void {
   }
   if (!has_name) {
     // Add a placeholder for the name.
-    context.AddLeafNode(NodeKind::IdentifierName, *context.position(),
+    context.AddLeafNode(NodeKind::IdentifierNameNotBeforeParams,
+                        *context.position(),
                         /*has_error=*/true);
```

Comment:
```suggestion
                        *context.position(), /*has_error=*/true);
```

The comment is probably stopping clang-format from rewrapping automatically.